src/commands/appSettings/filterDownloadAppSettings.ts

```diff
@@ -4,65 +4,61 @@
 *--------------------------------------------------------------------------------------------*/
 
 import * as vscode from 'vscode';
-import { DialogResponses, IActionContext } from "vscode-azureextensionui";
+import { IActionContext } from "vscode-azureextensionui";
 import { ext } from "../../extensionVariables";
 import { localize } from "../../localize";
 
 export async function filterDownloadAppSettings(context: IActionContext, sourceSettings: { [key: string]: string }, destinationSettings: { [key: string]: string }, destinationSettingsToIgnore: string[], destinationName: string): Promise<void> {
-    let suppressPrompt: boolean = false;
-    let overwriteSetting: boolean = false;
-    let overwriteSettingsToIgnore: boolean = false;
 
     const addedKeys: string[] = [];
     const updatedKeys: string[] = [];
     const userIgnoredKeys: string[] = [];
     const matchingKeys: string[] = [];
-    const securitySettingsIgnored: string[] = [];
-    const listOfSettingsToIgnore: string[] = ["AzureWebJobsStorage", "WEBSITE_CONTENTAZUREFILECONNECTIONSTRING", "WEBSITE_CONTENTSHARE"];
-
-    const yesToAll: vscode.MessageItem = { title: localize('yesToAll', 'Yes to all') };
-    const noToAll: vscode.MessageItem = { title: localize('noToAll', 'No to all') };
 
+    const listOfSettingsToIgnore: string[] = ["AzureWebJobsStorage", "WEBSITE_CONTENTAZUREFILECONNECTIONSTRING", "WEBSITE_CONTENTSHARE"];
+    const userChosenSettings: string[] = [];
+    const options: vscode.QuickPickItem[] = [];
     destinationSettingsToIgnore.length = 0;
```

Comment:
is `destinationSettingsToIgnore` still needed?

Reply:
I think so because it's the field containing the settings to ignore when deploying to function app.